joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/mips/MipsFunctionPass.scala

```diff
@@ -29,64 +29,68 @@ class MipsFunctionPass(currentProgram: Program,
   private val logger = LoggerFactory.getLogger(classOf[MipsFunctionPass])
 
   def resolveCallArguments(instruction: Instruction, callNode: CfgNodeNew): Unit = {
-    try {
-      highFunction
-        .getPcodeOps(instruction.getPcode.toList.last.getSeqnum.getTarget)
-        .asScala
-        .toList
-        .head
-        .getInputs
-        // First input is the instruction
-        // we have already handled it
-        .drop(1)
-        .zipWithIndex foreach {
-        case (input, index) =>
-          if (input.isRegister) {
-            var name = input.getHigh.getName
-            val high = input.getHigh
-            if (high != null && input.getDef != null && high.getName == "UNNAMED" && input.getDef != null && input.getDef.getInputs != null) {
-              val symbol = input.getDef.getInputs.toList.lastOption
-                .flatMap(x => Option(x.getHigh))
-                .flatMap(x => Option(x.getSymbol))
-              if (symbol.isDefined) {
-                name = symbol.get.getName
-              }
+    val target = instruction.getPcode.toList.head.getSeqnum.getTarget
+    //jal 0x00000000 and some jalr t9
+    if (highFunction
+          .getPcodeOps(target)
+          .asScala
+          .toList
```

Comment:
You can extract this into a `val pCodeOps = highFunction.getPcodeOps(target).asScala.toList` to avoid repeating it below.